src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -677,6 +678,7 @@ void GcsActorManager::ReconstructActor(const ActorID &actor_id, bool need_resche
           RAY_CHECK_OK(gcs_pub_sub_->Publish(
               ACTOR_CHANNEL, actor_id.Hex(),
               mutable_actor_table_data->SerializeAsString(), nullptr));
+          actor->UpdateAddress(rpc::Address());
```

Comment:
We can remove it directly, because actor is dead.

Reply:
will do!